wikidict/lang/en/__init__.py

```diff
@@ -47,7 +47,10 @@
     "en-past of",
     "en-simple past of",
     "en-superlative of",
+    "en-tpso",
```

Comment:
Might be good to add that same value to https://github.com/BoboTiG/ebook-reader-dict/blob/fd31b20cf26c945ce1079e220ac5a9241789e612/wikidict/render.py#L387

Reply:
ok